modules/commons/src/main/resources/schemas/neurosciencegraph/commons/selectors/v0.2.0.json

```diff
@@ -0,0 +1,176 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/"
```

Comment:
We might want to rename it to commons/selector/v0.2.0 to have it consistent with other schemas?

Reply:
If I remember correctly, @MFSY said to use the plural form for the ones where I did this. To be confirmed.